src/test/kotlin/no/nav/dagpenger/inntekt/v1/UklassifisertInntektApiTest.kt

```diff
@@ -258,6 +267,55 @@ class UklassifisertInntektApiTest {
         }
     }
 
+    @Test
+    fun `Post uklassifisert inntekt med feil redigert should return 200 ok`() = testApp {
```

Comment:
200 ok?

Reply:
Der har du rett i :D 

Se nå da, dette er gøy:

```suggestion
    fun `Post uklassifisert inntekt med feil redigert should return 400 ok`() = testApp {
```